src/NLU.DevOps.ModelPerformance/Utilities.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace NLU.DevOps.ModelPerformance
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Text;
+    using NLU.DevOps.ModelPerformance;
+
+    /// <summary>
+    /// This class contains utils function for performance calculations
+    /// and printing
+    /// </summary>
+    public static class Utilities
+    {
+       /// <summary>
+       /// Divides the dividend input by the diviso
+       /// </summary>
+       /// <param name="dividend"> The dividend in the division</param>
+       /// <param name="divisor"> The divisor in the division</param>
+       /// <returns>The division result</returns>
+        public static decimal Calculate(decimal dividend, decimal divisor)
+        {
+            return divisor != 0 ? decimal.Divide(dividend, divisor) : 0;
+        }
+
+        /// <summary>
+        /// Calculates the precision from a confusion matrix
+        /// </summary>
+        /// <param name="cm">confusion matrix metrics</param>
+        /// <returns>The precision result</returns>
+        public static decimal CalcPrecision(ConfusionMatrix cm)
```

Comment:
Consider making this an extension method on `ConfusionMatrix`:
```
public static double Precision(this ConfusionMatrix matrix) { ... }
```
So you can just run the following:
```
ConfusionMatrix matrix;
...
matrix.Precision()
```